GPU/GLES/GPU_GLES.cpp

```diff
@@ -297,6 +295,7 @@ static const CommandTableEntry commandTable[] = {
 	{GE_CMD_TEXFLUSH, 0},
 	{GE_CMD_TEXLODSLOPE, 0},
 	{GE_CMD_TEXSYNC, 0},
+	{GE_CMD_CLIPENABLE, 0},
```

Comment:
https://github.com/hrydgard/ppsspp/commit/b80c02c27259b9271261ccd0f26f73a749c538df#diff-5b998bdeb6be184246bb712047ce81f7R214

Maybe we'll never do it, but this was switched because of the way some games handle depth.  It needs to flush if we ever handle the clamping behavior properly, and it's hard for me to believe it frequently changes in most other games.

-[Unknown]